test/WebJobs.Script.Tests.Integration/NodeEndToEndTests.cs

```diff
@@ -442,10 +442,16 @@ public void ErrorFunction_DoesNotBringDownHost()
             request.Headers.Add("value", "TestValue");
 
             var id = Guid.NewGuid().ToString();
+            var metadata = new JObject()
+            {
+                { "m1", "AAA" },
+                { "m2", "BBB" }
+            };
             var input = new JObject()
             {
                 { "id", id },
-                { "value", "TestInput" }
+                { "value", "TestInput" },
+                { "metadata", metadata }
```

Comment:
Now this test is the same as the C# one - you can do the same in either language.